src/panel/app_panel.js

```diff
@@ -55,7 +55,16 @@ export default class AppPanel {
     }
 
     ReactDOM.render(<Menu />, document.querySelector('.react_menu__container'));
-    Telemetry.add(Telemetry.APP_START);
+    Telemetry.add(Telemetry.APP_START, null, null, {
+      'language': window.navigator.language,
+      'preferred_language': window.preferredLanguage,
+      'device_platform': window.navigator.platform,
+      'device_user_agent': window.navigator.userAgent,
+      'window_height': window.innerHeight,
```

Comment:
Meh, if we really need info about app display size, maybe we can just add two of them, because I doubt we'll ever do something different with device vs. window size.
Alternative: just sending the result of `Device.isMobile()`